app/qml/project/MMProjectHomeTab.qml

```diff
@@ -230,29 +116,39 @@ Item {
       topMargin: root.spacing
     }
 
+    activeProjectAlwaysFirst: true
+    projectsModel.activeProjectId: root.activeProjectId
+
+    list.section {
+      property: "ProjectIsActiveProject"
+      criteria: ViewSection.FullString
+      delegate: Column {
+        width: ListView.view.width
+
```

Comment:
Let's not use the default value of the `spacing` property of `Column`. Please set it explicitly.